app/src/components/ArtistPage.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import ArtistEvents from "./ArtistEvents";
+
+
+
+export default function ArtistPage() {
+
+    const { id } = useParams()
+    console.log(id);
+    const [artistId, setArtistId] = useState("");
```

Comment:
integer instead of string, can default to 0 since that doesn't match anyone or see if there is an undefined instead since that will work even better than just defaulting 0 for verification purposes

Reply:
Your request is outdated, only using const id with useparams. Artist id is removed allready